server/conn.go

```diff
@@ -37,50 +37,43 @@ type Conn struct {
 }
 
 var baseConnID uint32 = 10000
+var defaultServer *Server
 
 // NewConn: create connection with default server settings
```

Comment:
🎨 You should use godoc links, it would help people to navigate to the new resource.

```suggestion
// Deprecated: Use [Server.NewConn] instead.
```

Here also you can use pkgsite to check the difference